<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteManage</class>
 <widget class="QWidget" name="NoteManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sold Note Manage</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>791</width>
     <height>461</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Seller ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Seller Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Good ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Good Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Good Price</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Open </string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>530</y>
     <width>94</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Note</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteManage</receiver>
   <slot>onAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteManage</receiver>
   <slot>onDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteManage</receiver>
   <slot>onApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteManage</receiver>
   <slot>onResetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>NoteManage</receiver>
   <slot>onOpenClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>NoteManage</receiver>
   <slot>onItemChanged(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddClicked()</slot>
  <slot>onDeleteClicked()</slot>
  <slot>onApplyClicked()</slot>
  <slot>onResetClicked()</slot>
  <slot>onOpenClicked()</slot>
  <slot>onItemChanged(QTableWidgetItem*)</slot>
 </slots>
</ui>
